src/main/kotlin/racing/view/ResultView.kt

```diff
@@ -0,0 +1,40 @@
+package racing.view
+
+import racing.model.Car
+
+object ResultView {
+    private const val CAR_NAME_SEPARATOR = " : "
+    private const val ONE_BLOCK = "-"
+
+    fun output(numberOfTrials: Int, cars: List<Car>) {
+        (1..numberOfTrials).forEach {
+            println("trial$CAR_NAME_SEPARATOR$it")
+            outputForTrials(it, cars)
+        }
+    }
+
+    fun noticeWinner(winners: List<Car>) {
+        var winnersName = winners[0].name
+
+        for (i in 1 until winners.size) {
+            winnersName += ","
+            winnersName += winners[i].name
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val winnerNames = winners.joinToString(separator = ",") { it.name }
```

Reply:
간결하네요! 반영하였습니다. 의견 감사합니다.